package/doc/sphinx/source/documentation_pages/trajectory_transformations.rst

```diff
@@ -0,0 +1,69 @@
+.. Contains the formatted docstrings for the transformations located in 'mdanalysis/MDAnalysis/transformations'
+.. _transformations:
+
+**************************
+Trajectory transformations
+**************************
+
+.. module:: MDAnalysis.transformations
+
+The transformations submodule contains a collection of functions to modify the 
+trajectory. Coordinate transformations, such as PBC corrections and molecule fitting
+are often required for some analyses and visualization, and the functions in this 
+module allow transformations to be applied on-the-fly. 
+These transformation functions can be called by the user for any given
+timestep of the trajectory, added as a workflow using :meth:`add_transformations`
+of the :mod:`~MDAnalysis.coordinates.base` module, or upon Universe creation using
+the keyword argument `transformations`. Note that in the two latter cases, the 
+workflow cannot be changed after being defined.
+
+In addition to the specific arguments that each transformation can take, they also 
```

Comment:
An example with a more basic transformation that does not require arguments may fit well here. It can be something extremely basic such as:

```python
def up_by_2(ts):
    """
    Translate all coordinates by 2 Å up along the Z dimension.
    """
    ts.positions = ts.positions + np.array([0, 0, 2], dtype=np.float32)
    return ts
```